demo/mobile/src/app/config.js

```diff
@@ -11,21 +11,17 @@ function createMainAppRoute($stateProvider, $urlRouterProvider) {
       abstract: true,
       templateUrl: 'app/main.tpl.html',
       resolve: {
-        profileData: function(userClient) {
-          return userClient.getProfile();
-        },
-        syncManagers: function(syncPool, profileData) {
-          return syncPool.syncManagerMap(profileData);
-        },
-        workorderManager: function(syncManagers) {
-          return syncManagers.workorders;
-        },
-        workflowManager: function(syncManagers) {
-          return syncManagers.workflows;
-        }
+        // profileData: function(userClient) {
+        //   return userClient.getProfile();
+        // },
+        // workorderManager: function(syncManagers) {
+        //   return syncManagers.workorders;
+        // },
+        // workflowManager: function(syncManagers) {
+        //   return syncManagers.workflows;
+        // }
```

Comment:
Yeah. I'm having changes as part of another ticket and PR and wanted this to be merged in order to bring that in.